samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/SelectAllTextBehaviorPage.xaml

```diff
@@ -0,0 +1,85 @@
+				<Label FontAttributes="Italic" Text="Entry with the effect, when focussed all text will be selected" />
+				<Frame BackgroundColor="{StaticResource SoftFrameBackgroundColor}" HasShadow="False">
+					<Entry
+						PlaceholderColor="{StaticResource DarkLabelPlaceholderColor}"
+						Text="https://github.com/xamarin/XamarinCommunityToolkit"
+						TextColor="{StaticResource DarkLabelTextColor}">
+						<Entry.Behaviors>
+							<mct:SelectAllTextBehavior />
+						</Entry.Behaviors>
+					</Entry>
+				</Frame>
+			</VerticalStackLayout>
+
+			<Label
+				FontAttributes="Bold"
+				Text="Editor"
+				TextColor="{StaticResource DarkLabelTextColor}" />
+
+			<VerticalStackLayout>
+				<Label FontAttributes="Italic" Text="Editor without the effect, when focussed no text will be selected" />
+				<Frame BackgroundColor="{StaticResource SoftFrameBackgroundColor}" HasShadow="False">
+					<Editor PlaceholderColor="{StaticResource DarkLabelPlaceholderColor}" TextColor="{StaticResource DarkLabelTextColor}">
+						<Editor.Text>
+							Yeah, but your scientists were so preoccupied with whether or not they could, they didn't stop to think if they should. Must go faster... go, go, go, go, go! You know what? It is beets. I've crashed into a beet truck. Is this my espresso machine? Wh-what is-h-how did you get my espresso machine?
+							Just my luck, no ice. I gave it a cold? I gave it a virus. A computer virus. Must go faster... go, go, go, go, go! God creates dinosaurs. God destroys dinosaurs. God creates Man. Man destroys God. Man creates Dinosaurs. Yeah, but your scientists were so preoccupied with whether or not they could, they didn't stop to think if they should.
+						</Editor.Text>
+					</Editor>
+				</Frame>
+			</VerticalStackLayout>
+
+			<VerticalStackLayout>
+				<Label FontAttributes="Italic" Text="Editor with the effect, when focussed all text will be selected" />
+				<Frame BackgroundColor="{StaticResource SoftFrameBackgroundColor}" HasShadow="False">
+					<Editor PlaceholderColor="{StaticResource DarkLabelPlaceholderColor}" TextColor="{StaticResource DarkLabelTextColor}">
+						<Editor.Behaviors>
+							<mct:SelectAllTextBehavior />
+						</Editor.Behaviors>
+						<Editor.Text>
+							Yeah, but your scientists were so preoccupied with whether or not they could, they didn't stop to think if they should. Must go faster... go, go, go, go, go! You know what? It is beets. I've crashed into a beet truck. Is this my espresso machine? Wh-what is-h-how did you get my espresso machine?
+							Just my luck, no ice. I gave it a cold? I gave it a virus. A computer virus. Must go faster... go, go, go, go, go! God creates dinosaurs. God destroys dinosaurs. God creates Man. Man destroys God. Man creates Dinosaurs. Yeah, but your scientists were so preoccupied with whether or not they could, they didn't stop to think if they should.
+						</Editor.Text>
+					</Editor>
+				</Frame>
+			</VerticalStackLayout>
+		</VerticalStackLayout>
```

Comment:
Let's avoid nesting multiple VerticalStackLayouts inside of a VerticalStackLayout. 

Let's either use a `Grid`, or a `FlexLayout`, or edit each control's `Padding` + `Margin` + `Spacing` to fine-tune the groupings. 

https://docs.microsoft.com/en-us/xamarin/xamarin-forms/deploy-test/performance#choose-the-correct-layout
> ... don't attempt to reproduce the appearance of a specific layout by using combinations of other layouts, as this results in unnecessary layout calculations being performed. For example, don't attempt to reproduce a Grid layout by using a combination of StackLayout instances.